packages/framework/src/Controller/Admin/ProductController.php

```diff
@@ -165,13 +165,16 @@ public function editAction(Request $request, $id)
         if ($form->isSubmitted() && $form->isValid()) {
             $this->productFacade->edit($id, $form->getData());
 
-            $this->getFlashMessageSender()->addSuccessFlashTwig(
-                t('Product <strong>{{ product|productDisplayName }}</strong> modified'),
-                [
-                    'product' => $product,
-                ]
-            );
-            return $this->redirectToRoute('admin_product_edit', ['id' => $product->getId()]);
+            $this->getFlashMessageSender()
```

Comment:
this behavior change is not explained in the US. I didn't find a criteria that would say "if the product is successfully edited, redirect to list". Am I right?

Reply:
The same applies for a new product

Reply:
Well, reason for this was that product stuff in admin behaved different way than the others. I believe this is right and will discuss during BV, are you ok with that?

Reply:
sure

Reply:
@PetrHeinz our current business validator agreed with me